assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/TicketPriceCalculatorModal.tsx

```diff
@@ -14,28 +14,20 @@ const TicketPriceCalculatorModal: React.FC = () => {
 		onCloseModal,
 		dataState: { reset, prices, ticket },
 	} = useTPCContext();
-	const resetButton = useResetButtonProps({ reset });
-	const submitButton = useSubmitButtonProps({ onCloseModal, prices });
+	const resetButtonProps = useResetButtonProps({ reset });
+	const submitButtonProps = useSubmitButtonProps({ onCloseModal, prices });
 
 	return (
 		<Modal
-			bodyStyle={{ padding: 0 }}
-			cancelButtonProps={resetButton}
-			cancelText={__('Reset')}
-			centered
-			maskClosable
-			okButtonProps={submitButton}
-			okText={__('Submit')}
-			onCancel={onCloseModal}
-			onOk={onCloseModal}
+			isOpen={true}
+			onClose={onCloseModal}
+			className='ee-tpc__modal'
+			bodyClassName='ee-tpc__body'
```

Comment:
```suggestion
			className='ee-tpc-modal'
			bodyClassName='ee-tpc-modal__body'
```

FYI this is following a CSS naming schema called BEM

here's some links about it:

- [short quick read](https://codeburst.io/understanding-css-bem-naming-convention-a8cca116d252)

- [a little more in depth article](https://seesparkbox.com/foundry/bem_by_example)

- [another great article from same site](https://seesparkbox.com/foundry/bem_css_organization)

but you can skip getting into all of that and leave the UI polish and such to Vadim and I if you like